code/Model/SiteTree.php

```diff
@@ -2009,7 +2009,8 @@ public function getCMSFields()
                 'SilverStripe\\CMS\\Controllers\\CMSMain.NEWPAGE',
                 'New {pagetype}',
                 array('pagetype' => $this->i18n_singular_name())
-            )));
+            )))
+            ->addExtraClass(($this->isHomePage()?'homepage-warning':''));
```

Comment:
Can you run a linter over this PR please?

Reply:
This was resolved but it looks like the linter config might be wrong? There should be spaces around the ternary delimiters here. Additionally there are some extra brackets in there.